.github/workflows/test-action.yml

```
@@ -0,0 +1,118 @@
+name: Test GitHub Action
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    types: [ opened, synchronize, reopened, ready_for_review ]
+
+jobs:
+  test-defaults:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Test GitHub Action
+        uses: ./
+        with:
+          filepath: test/asyncapi.yml
+      - name: Assert GitHub Action
+        run: |
+          echo "Asserting GitHub Action"
+          if [ -f "./output/asyncapi.md" ]; then
+            echo "Files exist"
+          else
+            echo "Files do not exist:- ./output/asyncapi.md"
+            echo "Action failed"
+            exit 1
+          fi
```

```suggestion
          echo "Listing all files"
          ls -R
          echo "Asserting GitHub Action"
          if [ -f "./output/asyncapi.md" ]; then
            echo "Files exist"
          else
            echo "Files do not exist:- ./output/asyncapi.md"
            echo "Action failed"
            exit 1
          fi
```

please modify all assertions to have it